code-challenges.js

```diff
@@ -1,51 +1,45 @@
 // ASSESSMENT 4: JavaScript Coding Practical Questions
-
 // --------------------1) Create a function that takes in an array. Each time the function is run, remove the first item from the array and shuffle the remaining content. If the array is empty, return "The array is empty."
-
 var collections = ["purple", "blue", "green", "yellow", "pink"]
 // Expected output example1 (can be a different order): ["yellow", "blue", "pink", "green"]
 // Expected output example2 (can be a different order): ["blue", "green", "pink"]
 // Expected output example3 (can be a different order): ["pink", "green"]
-
-
-
-
-
+//
+const minus1 = (array) => {
+  var collections = array.shift(0)
+  for(let i = 0; i < 0; i++){
+    return array
+  }
+}
+console.log(minus1(collections))
 // --------------------2) Create a function that takes in an array of numbers and returns the sum of all the numbers cubed.
-
 var cubeAndSum1 = [2, 3, 4]
 // Expected output: 99
 var cubeAndSum2 = [0, 5, 10]
 // Expected output: 1125
-
-
-
-
-
+const cubed = (array) => {
+  let cubed = cubeAndSum1.map(value =>
+     math.cbrt()
+  )
+}
+cosole.log(cubed(cubeAndSum1))
```

Comment:
I'm also running into issues getting this to run correctly. The console log is incorrect and it looks like the math needs to be capitalized. Outside of that, if i make those changes, I can't seem to get the code to produce anything either.